metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -646,10 +646,13 @@ def emit_sheets_as_charts(self, workbook: Dict) -> Iterable[MetadataWorkUnit]:
 
             if sheet.get("path", ""):
                 sheet_external_url = f"{self.config.connect_uri}#/site/{self.config.site}/views/{sheet.get('path', '')}"
-            else:
+            elif sheet.get("containedInDashboards"):
                 # sheet contained in dashboard
                 dashboard_path = sheet.get("containedInDashboards")[0].get("path", "")
                 sheet_external_url = f"{self.config.connect_uri}/t/{self.config.site}/authoring/{dashboard_path}/{sheet.get('name', '')}"
+            else:
+                # hidden or viz-in-tooltip sheet
+                sheet_external_url = f"{self.config.connect_uri}"
```

Comment:
I don't have a strong opinion here. I thought it would be useful to track the actual tableau instance, but we also have the path to the site/project. 
We can also set it to `None` and the link won't be rendered. WDYT?

Reply:
I think, let's set this to None for now, rather than taking user on a page where he has no clue how to find respective sheet.

Reply:
Changed in https://github.com/linkedin/datahub/pull/4231/commits/ed787c74495066c53d6d02a550cee84ae4e2bfed